packages/mask/src/utils/utils.ts

```diff
@@ -53,3 +55,19 @@ export function regexMatch(input: string, pattern: RegExp, index: number | null
     }
     return r[index]
 }
+
+export async function attachNextIDToProfile(nextID: ProfileInformationFromNextID) {
+    const whoAmI = await Services.Settings.getCurrentPersonaIdentifier()
+
+    if (!nextID?.fromNextID || !nextID.linkedPersona || !whoAmI) return
+    const [rpc, emit] = batch(Services.Identity)
+    nextID.linkedTwitterNames?.forEach((x) => {
+        const newItem = {
+            ...nextID,
+            nickname: x,
+            identifier: ProfileIdentifier.of('twitter.com', x).expect(`${x} should be a valid user id`),
+        }
+        rpc.attachNextIDPersonaToProfile(newItem, whoAmI)
```

Comment:
if you do not need the return value, you can also use `notify` from `async-call-rpc`. It will drop the return value.

Reply:
notify and batch can be used together. if you need it to improve the performance.